sippy-ng/src/tests/TestAnalysis.js

```diff
@@ -204,8 +268,33 @@ export function TestAnalysis(props) {
   })
 
   let index = 0
+  variants.forEach((variant) => {
+    byVariantChart.datasets.push({
```

Comment:
Can this be put into the same loop on line 258?

Reply:
it can, good idea. Updated.